app/assets/javascripts/components/overview/course_edit_toggle.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import CourseActions from '../../actions/course_actions.js';
+import uuid from 'uuid';
+
+const CourseEditToggle = React.createClass({
```

Comment:
This looks like it duplicates a lot of the logic from YesNoSelector, which was recently extracted. Could you use that instead? See PrivacySelector, for example.